### ISSUE TITLE
ggplot axis text issue

### ISSUE THREAD

**Issue by jakeybob** [tylermorganwall/rayshader#59]
Not sure if this is related to #53 or #26 but since the update yesterday I've encountered an issue whereby specifying `axis.text.y = element_blank()` causes the 3D plotting to flatten . e.g.

```R
library(rayshader)
library(reshape2)
library(ggplot2)

ggvolcano <- volcano %>% 
  melt() %>%
  ggplot() +
  geom_tile(aes(x = Var1, y = Var2, fill = value)) +
  geom_contour(aes(x = Var1, y = Var2, z = value), color='black') +
  scale_x_continuous("X", expand = c(0, 0)) +
  scale_y_continuous("Y", expand = c(0, 0)) +
  scale_fill_gradientn("Z", colours = terrain.colors(10)) +
  coord_fixed() +
  theme_bw() + 
  theme(axis.title = element_blank(),
        axis.text.x = element_blank(),
        axis.text.y = element_blank(), # try commenting out this line
        axis.ticks = element_blank(),
        panel.grid = element_blank(),
        panel.border = element_blank())

plot_gg(ggvolcano, width = 6, height = 4, raytrace = FALSE, preview = FALSE, scale = 500)
```
...plots the following
![3dYon](https://user-images.githubusercontent.com/304817/60724004-28e5bc00-9f2d-11e9-8652-99c68e0530f1.png)

...and commenting out the `axis.text.y` line gives
![3dYoff](https://user-images.githubusercontent.com/304817/60724031-4155d680-9f2d-11e9-9b03-a74d90de74c5.png)

(note, will also render as flat when using `axis.text = element_blank()` to set both axes.)

As a workaround, I'm using e.g. `axis.text.y = element_text(colour = "white")`, but thought I should flag this up as an issue. 😃 

`sessionInfo()`:
```R
R version 3.5.1 (2018-07-02)
Platform: i386-w64-mingw32/i386 (32-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.1252  LC_CTYPE=English_United Kingdom.1252   
[3] LC_MONETARY=English_United Kingdom.1252 LC_NUMERIC=C                           
[5] LC_TIME=English_United Kingdom.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ggplot2_3.2.0    reshape2_1.4.3   rayshader_0.11.2

loaded via a namespace (and not attached):
 [1] jsonlite_1.6            rstudioapi_0.10         magrittr_1.5            gtable_0.3.0           
 [5] hms_0.4.2               webshot_0.5.1           bmp_0.3                 pillar_1.4.2           
 [9] htmltools_0.3.6         foreach_1.4.4           progress_1.2.2          stringr_1.4.0          
[13] readbitmap_0.1.5        later_0.8.0             manipulateWidget_0.10.0 dplyr_0.8.1            
[17] htmlwidgets_1.3         tidyselect_0.2.5        plyr_1.8.4              miniUI_0.1.1.1         
[21] knitr_1.23              igraph_1.2.4.1          iterators_1.0.10        mime_0.7               
[25] pkgconfig_2.0.2         R6_2.4.0                shiny_1.3.2             imager_0.41.2          
[29] digest_0.6.20           xfun_0.8                colorspace_1.4-1        stringi_1.4.3          
[33] yaml_2.2.0              lazyeval_0.2.2          crosstalk_1.0.0         codetools_0.2-15       
[37] tibble_2.1.3            compiler_3.5.1          withr_2.1.2             doParallel_1.0.14      
[41] tiff_0.1-5              xtable_1.8-4            munsell_0.5.0           Rcpp_1.0.1             
[45] png_0.1-7               parallel_3.5.1          assertthat_0.2.1        rgl_0.100.24           
[49] prettyunits_1.0.2       jpeg_0.1-8              tools_3.5.1             httpuv_1.5.1           
[53] scales_1.0.0            crayon_1.3.4            glue_1.3.1              purrr_0.3.2            
[57] rlang_0.4.0.9000        promises_1.0.1          grid_3.5.1      
```

(this also happened on my Mac at home last night, so not Windows specific)

**Comment by tylermorganwall** [comment]
This should be fixed in 8144c990838a69d4565753407aadd44c2ec99be5

**Comment by jakeybob** [comment]
Thanks – enjoying the package and really appreciating the quick fixes! 😃
